tools/expressionmerge.py

```diff
@@ -35,7 +36,7 @@
     base, ext = os.path.splitext(f)
     delimiter = ';' if ext == '.csv' else '\t'
 
-    with gzip.open(f, 'rb') as csvfile:
+    with io.TextIOWrapper(io.BufferedReader(gzip.open(f))) as csvfile:
```

Comment:
@kostko, is there a better solution? The problem is that users upload .csv files with windows (/r/n), linux (/n) or mac (/r) line endings and that simply iterating through the file (gzip.open) fails for .csv files generated with Excel on Macs.